my-app/src/FactContainer.js

```diff
@@ -5,19 +5,7 @@ import './FactContainer.css';
 
 
 function FactContainer({facts, deleteFact, favChange}){
-    FactContainer.propTypes = {
-      facts: PropTypes.arrayOf(
-      PropTypes.shape({
-        name: PropTypes.string.isRequired,
-        text: PropTypes.string.isRequired,
-        favorite: PropTypes.bool.isRequired,
-        id: PropTypes.number.isRequired
-    }))
-        .isRequired,
-        deleteFact: PropTypes.func.isRequired,
-        favChange: PropTypes.func.isRequired
-  }
-  
+   
     const factCards = facts.map(fact => {
       return (
         <Fact
```

Comment:
If you wanted to pass fewer props, you could pass the whole fact object deconstructed. Yours works great as is, but you could pass it as, {...fact} and pull out properties within the params fact component :)